### ISSUE TITLE
Test use cases (Automated and Manual)

### ISSUE THREAD

**Issue by KPHowley** [hackforla/tdm-calculator#207]
### Overview
In order to ensure the calculator still works as intended we need to test use cases before deploying a new release.  This should be an automated process.  In the meantime we have a manual testing procedure, that must be done before every pull request.

### Action Items
**Manual Test**
Test the results of the calculator against the following results which were initially verified as the DOT's intended outcomes:

- [ ] Barrington : [ Barrington Saved PDF](https://drive.google.com/file/d/1V9TEkzSr0cckBY2jqu1lfwntev6wkaA2/view?usp=sharing) 
- [ ] Fountain: [Fountain Saved PDF]( https://drive.google.com/file/d/1n-1dcEUlnkKAFAthbgjcmiy9Zeq9UE1c/view?usp=sharing)
- [ ] Victory [Victory Saved PDF]( https://drive.google.com/file/d/1kSUnU2qjS4Ntyv226cEAnF-O8i3lMsrR/view?usp=sharing)
- [ ] Beatrice [Beatrice Saved PDF](https://drive.google.com/file/d/1YnwJA49s46SU8symklAcR2LNbGAVpqaF/view?usp=sharing)
- [ ] Clarendon: [Clarendon Saved PDF](https://drive.google.com/file/d/1J_FmGk-KQv5h_DAvZrKX3K878Iy9TVB9/view?usp=sharing)
- [ ] Jewish Family Services (Fairfax): [Fairfax Saved PDF](https://drive.google.com/file/d/1GSrxSUQD4F1ovUys2C0Zhc9q7A36C-_1/view?usp=sharing)

**Automated Testing** 
???

### Resources/Instructions
Saved projects list: http://tdm-calc-staging.herokuapp.com/projects

Currently the Calculator re-runs all calculations when a saved projects is selected from the projects page. Verifying this against the results saved in the above PDF's should then verify that all calculations are still working as intended.

**Comment by mmorgs** [comment]
@KPHowley - is this something we could add to the wiki? maybe it'd be easier to find that way. thoughts?

**Comment by ExperimentsInHonesty** [comment]
@fyliu 
Is Jest going to accomplish the testing required in this issue or is it a backend test using other testing framework?

**Comment by fyliu** [comment]
@ExperimentsInHonesty I think Jest will work for this. It should be a matter of taking the rules input object and feeding it to the calculation engine and reading the results.

**Comment by ExperimentsInHonesty** [comment]
@KPHowley will make a template and slack it @joelparkerhenderson l for advice
@Bradley is doing research on React Testing Library
@nclairesays is doing research on Cypress
@fyliu is doing research on Selenium

**Comment by KPHowley** [comment]
A new Issue was created for automated testing research - #231  closing this issue
